src/cpu/aarch64/brgemm/jit_brdgmm_kernel.hpp

```diff
@@ -116,9 +117,7 @@ struct jit_brdgmm_kernel_base_t : public jit_generator {
     int tail_length() { return n_block1_tail() % simd_w_; }
     bool is_fma_embd() { return brg.is_f32; }
     bool is_fast_vnni_int8() { return is_fast_vnni_int8(brg); }
-    int vnni_substep() {
-        return brg.isa_impl == sve_256 && (brg.is_bf16 || brg.is_f16) ? 2 : 1;
-    }
+    int vnni_substep() { return brg.isa_impl == sve_256 && brg.is_f16 ? 2 : 1; }
```

Comment:
is `f16` even allowed here? I only see support for `f32`, `int8`, and `bf16` (now).